### DISCUSSION TITLE
Unable to get a property from a blueprint actor

### DISCUSSION THREAD

**Discussion by czcz1024** [nxrighthere/UnrealCLR#142] [Help] [answered]
public static void AppendString(ObjectReference self)
{
    Actor blueprintActor = self.ToActor<Actor>();
    string v="";
    var r= blueprintActor.GetText("vstr", ref v);
    Debug.AddOnScreenMessage(-1, 10.0f, Color.AliceBlue, "get "+r+",v "+v);
    
}
i have a function like this
i create an actor and add a var named "vstr",and drag it in to sence
and in lev blueprint run  Excute Managed Function and pass an actor instance.
but GetText return false
![image](https://user-images.githubusercontent.com/2842888/119958367-91ccf600-bfd5-11eb-8b23-241e1a031bc9.png)
![image](https://user-images.githubusercontent.com/2842888/119958639-ce98ed00-bfd5-11eb-98cb-33865f09edbe.png)

what's wrong?

**Comment by nxrighthere** [comment, marked as answer]
You are trying to obtain a text property using `blueprintActor.GetText()` function. The property itself should be set to `Text` type instead of `String`.

**Reply by czcz1024** [reply]
yes, thank you

**Reply by nxrighthere** [reply]
String properties for actors and components exposed in https://github.com/nxrighthere/UnrealCLR/commit/6b85ca17dc49f74e53674bec8e619245d7125e5b.
